src/engine/enginebuffer.cpp

```diff
@@ -171,8 +171,7 @@ EngineBuffer::EngineBuffer(QString group, UserSettingsPointer pConfig,
     m_pRepeat = new ControlPushButton(ConfigKey(m_group, "repeat"));
     m_pRepeat->setButtonMode(ControlPushButton::TOGGLE);
 
-    // Sample rate
-    m_pSampleRate = new ControlProxy("[Master]", "samplerate", this);
+    m_pMasterSampleRate = new ControlProxy("[Master]", "samplerate", this);
```

Comment:
m_pKeylockEngine is also bound to "[Master]" and doesn't contain a prefix in its name. Do we really need to add "Master" just for this member?